experimenter/experimenter/templates/components/change_card.html

```diff
@@ -0,0 +1,77 @@
+<div x-data="{expand:false}" :class="{'bg-white border border-gray-100 shadow-md rounded-md': expand }"  class="p-2">
```

Comment:
I believe one is for desktop and one is for mobile version, would suggest adding a comment to keep the them separate What css classes have been applied

Reply:
This was for the expanded view of the change, the padding remains the same but the rest of the styling is added when the card expands, I'll add a comments that specify behaviour if dependant on alpine states